docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,15 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service Has Been Forced to Roll Back to Zero (CB90067)
```

Comment:
The title is a bit long and wordy - perhaps "Rollback To Zero" may be snappier?

Reply:
Should I do that for both titles or just the title for cmos-health-checks.adoc?

Reply:
Both please, let's keep them as consistent as we can :)